README.adoc

```diff
@@ -312,6 +312,8 @@ For Gradle, add the following dependency:
 ====
 [source,java]
 ----
+implementation files('../library/target/multi-module-library-0.0.1-SNAPSHOT.jar')
```

Comment:
This looks bad to me (fragile). Why did we switch it from `implementation project(':library')`?

Reply:
The `implementation project(':library')` method wasn't building for me. I did some reading. Apparently, the Gradle folks prefer the `files` method.

Reply:
I got it to work. If i run the build from within the application project, it fails on that `library` line. If I run the build from the parent directory, it works. The instructions in the README say to run it from the parent directory, so OK. I want them to work in isolation, too, but so it goes.

Reply:
I set it back to what it was.

Reply:
FWIW, I've never seen a recommendation to use `files`. You should use a `project` dependency as it allows Gradle to determine the relationship between the projects and the resulting graph of tasks that need to be executed. If you want to build a task in an individual project you should be able to do that from the root directory by specifying the full path of the task. Gradle will then automatically figure out what, if any, other tasks need to be run first.